src/appshell/view/appmenumodel.cpp

```diff
@@ -470,11 +470,11 @@ MenuItemList AppMenuModel::makeTupletsItems()
 MenuItemList AppMenuModel::makeMeasuresItems()
 {
     MenuItemList items {
-        makeMenuItem("insert-measure"),
-        makeMenuItem("insert-measures"),
+        makeMenuItem("insert-measures-after-selection", qtrc("notation", "Insert after selection…")),
+        makeMenuItem("insert-measures", qtrc("notation", "Insert before selection…")),
         makeSeparator(),
-        makeMenuItem("append-measure"),
-        makeMenuItem("append-measures")
+        makeMenuItem("insert-measures-at-start-of-score", qtrc("notation", "Insert at start of score…")),
+        makeMenuItem("append-measures", qtrc("notation", "Insert at end of score…"))
```

Comment:
and "Append to score" or "Append at end of score")?

Reply:
I'm passing these questions to @bkunda, who made the design. 🙂

Reply:
Or " Add measures ...„ for all 4 strings

Reply:
Just to clarify: is the question here what the menu item labels should be in the "Add" > "Measures" menu?

Reply:
Yes, and only that. The functionality is not questioned and very welcome

Maybe we could add "insert to beginning is scetion" and "append to end of section", but that is not as important as the other additions at all.

Reply:
In 3.x we had "Append measures" (to mean Append measures at end of score) and "Insert measures" (to mean insert before selection), I'd rather keep the new texts as close as possible to those.
So "Insert measures at start of score", Insert measures before selection", "Append/Insert measures to/after selection" (not really sure about the best wording here), "Apend measures to end of score"

Reply:
This is what I put in my design spec. The idea was to align the labels in this menu with the new dropdown feature: 

<img width="1137" alt="Screenshot 2022-02-01 at 6 15 23 pm" src="https://user-images.githubusercontent.com/86290556/151927283-d363de88-b347-4a44-9066-6060ce6c1555.png">

I tried to avoid the word "Append". It of course makes sense, but is generally not a super-frequently used word in English, and I reasoned that "Insert+location" communicated essentially the same message. 

In saying this, I'm wondering in my own design whether the word "Insert" is even necessary. The action sequence is, after all, `Add`>`Measures`>`After selection` (or whatever other location designation).

Reply:
I'd swap "Insert after selection" and "Insert before selection", seems more logical (and maybe rename the former to "Append to selection"). Bit in the contect of the "Add > Measure" indeed Insert and Append are not needed, In the shortcuts menu though this is different.
So "Add" > "Measure" > "Before selection" / "After Selection" / "At start of score" / "At end of score" seems good to me

Reply:
Also the menus needs to show the shortcuts

Reply:
What about single measures?

Reply:
Ok I've just checked the implementation in master again and I think can better see the scope of the issue now (my apologies, I don't think I was completely understanding what the issue was before).

The "Add" menu is as I've designed it, and I think it works well: 
<img width="353" alt="Screenshot 2022-02-01 at 8 46 23 pm" src="https://user-images.githubusercontent.com/86290556/151946194-c99d70e8-6acd-40c5-8239-ec419857d035.png">

The `+` menu, however, has inconsistent terminology and functionality:

<img width="469" alt="Screenshot 2022-02-01 at 8 45 42 pm" src="https://user-images.githubusercontent.com/86290556/151946345-9801b2e5-7fc9-4b54-acb5-14ce165f0fc6.png">

My recommendation would be that we align the options in the `+` menu with what we have in the `Add` menu. It would therefore make sense to retain the use of the word "insert" as this provides better context for the options in the `+` menu. 

All menu items trigger the "Number of measures to insert" dialog, which comes pre-filled with a single measure (this is how single measures can be entered). 

Shortcuts do need tweaking as well now, because the dialog that gets triggered doesn't delineate between measures inserted at the selection point, and measures inserted at the end of the score. 

Since the present PR has already been merged, I'll raise these tweaks as a separate issue.

Reply:
That, for single measures, would be more complicated then before, a simple 'Ins' inserted a single measure before the selection and a Ctrl+Ins appended one measure to the end of the score, very simple and efficient, now is a much more involved

Reply:
Sorry, I didn't mean to suggest we would change the way the shortcuts work (I also quite like how easy they make inserting a single measure). I only meant to suggest that the dialog no longer gives adequate information for the user to know where they are entering bars (on `⌥⇧B` command).

Reply:
I see. Still the Insert/Append one measure seem missing from the menu?

Reply:
I'm afraid this was a by-product of our efforts to both clarify the intended result of the menu actions while also simplifying the language.

The problems in MS3 are that:
- "Append" can be an impenetrable term for a lot of people
- The effect of "Insert measures" and "Insert one measure" never made clear _where_ the inserted measure would be applied (I.e. _before_ or _after_ the selection point)? The user had to actually click on the label to work out where the new measure was inserted (and because the new bar(s) never got highlighted, it was doubly confusing trying to work out the whereabouts of the new measure(s)).
 
In trying to move away from the word "Append", while also making it very clear in the label _where_ the inserted measure would be applied, some simplification in the logic was required, lest we end up with a whole string of wordy options ("insert one measure before selection point", "insert one measure after selection point", "insert measures before selection point", "insert measures after selection point", "insert one measure at the end of the score"...etc, etc.). This included moving away from the distinction between inserting just one measure, and inserting multiple measures. We rationalised that this wouldn't be such a bad trade-off, however, as inserting measures is typically an action that doesn't usually take place with any tremendous frequency in the course of creating a score (It's usually a much less frequent function than, say, inserting notes and rests, or adding dynamics or articulation marks to a score).

A (hopefully) nice result of this is that the user now has additional options for specifying where they want the inserted measures to appear ("at start of score" and "after selection point" are new). On top of this, we've doubled the number of access points for inserting measures, exposing this functionality in what we hope will be a much more visible way. Obviously I'll be testing all this in user testing in the near future, but we are hoping all these revisions make for a less, rather than more, complicated user experience.

Reply:
I agree that "append" is better avoided.